e2e/wdio/README.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+WDIO Automation
+
+# Getting Started
+
+Prereq - app to be tested must already be deployed and has access to experience editor
+
+1.	Run `$npm i`
+2.	Update .env fields BASEURL, USER_NAME, PASSWORD, and APPNAME or can provide values from cmdline
+3.  To run tests: 
+    - Run single spec file - `set "APPNAME=JssNextWeb" & set "PASSWORD=b" && npx wdio wdio.conf.js --baseUrl https://ssge2ecm --spec ./test/specs/experience-editor.spec.js`
+    - Run suite as defined in wdio.conf.js - `set "APPNAME=etoenextjs" & set "PASSWORD=b" && npx wdio wdio.conf.js --baseUrl https://cm.jssdev.localhost --suite ee`
+4. To run setup script for new fresh environments `set "APPNAME=etoenextjs" & set "PASSWORD=b" && npx wdio wdio.conf.js --baseUrl https://cm.jssdev.localhost --suite setup`. This will: 
```

Comment:
Is there a reason the setup script is a test suite (e.g. are we testing the setup itself)? Or could this just be a regular JS file/function that's hooked up to another package.json script (e.g. "setup")?